src/main/resources/data/configuration/prefixes/prefix.json

```diff
@@ -1,5 +1,5 @@
 {
   "id": "db9f5d17-0ca3-4d14-ae49-16b63c8fc083",
-  "name": "poNumberPrefix",
-  "description": "Description"
+  "name": "pref",
```

Comment:
I don't understand why this is here... This is a module release PR, we should not be making changes like this in the context of this PR anyway.

Reply:
Oh, I see now - you were just pulling in changes that have been made since your created this PR...  I think if you rebased it would be more clear

Reply:
Thanks, I will take this into account.